Makefile

```diff
@@ -0,0 +1,33 @@
+.PHONY: requirements
+requirements:
+	pip install -r requirements.txt
+
+.PHONY: lint
+lint: requirements
```

Comment:
This is a small nitpick, but both `black` and `cf-lint` are linters, so `make black` and `make lint` is a bit confusing.

Can we rename this one to `make cf-lint`?